db/seeds.rb

```diff
@@ -27,6 +27,104 @@
 GradeLevel.create "title" => "College Senior", "level" => 16, "abbreviation" => "R"
 GradeLevel.create "title" => "College 5th-Year", "level" => 17
 
+# Help Texts
+HelpText.create({
+  "object_class" => "Participant", 
+  "attribute_name" => "household_size",
+  "hint" => "Include yourself in this count",
+  "audience" => "Participant"
+})
+HelpText.create({
+  "object_class" => "Mentor", 
+  "attribute_name" => "household_size",
+  "hint" => "Include the student in this count",
+  "audience" => "Mentor"
+})
+HelpText.create({
+  "object_class" => "Participant",
+  "attribute_name" => "immigrant",
+  "title" => "Do you identify yourself as an immigrant to the United States?",
+  "audience" => "Participant"
+})
+HelpText.create({
+  "object_class" => "Mentor",
+  "attribute_name" => "immigrant",
+  "title" => "Identifies as an immigrant to the United States",
+  "audience" => "Mentor"
+})
+HelpText.create({
+  "object_class" => "Participant",
+  "attribute_name" => "first_generation",
+  "title" => "Does one or more of your parents/legal guardians hold a bachelor’s/4 year degree?",
```

Comment:
![screenshot from 2016-04-02 13 16 27](https://cloud.githubusercontent.com/assets/8908184/14228799/262240c6-f8d5-11e5-8a92-f16844647da3.png)
The label for `immigrant` doesn't get displayed, however, for `first_generation` it does